### ISSUE TITLE
[ #2236 related; VS Code] Disable all of Pyright's hover features

### ISSUE THREAD

**Issue by octanervoc** [microsoft/pyright#3617]
**Is your feature request related to a problem? Please describe.**  
I do not have much RAM available and Pylance consumes it all whenever I use it. I need to rely on Jedi for basic hover features, but it does not give some warnings such as missing function parameters. Pyright is very fast and does not impact my memory usage, so it's my go-to choice for "linting". However, as mentioned in issue [#2236](https://github.com/microsoft/pyright/issues/2236), this means that Jedi's hover overlaps with Pyright. 

**Describe the solution you'd like**  
Is it possible to implement an option to disable all of Pyright's hover features in favor of Jedi? The documentation very explicitly states that `pyright.disableLanguageServices` only "Disables all language services except for 'hover'." 

**Additional context**  
![image](https://user-images.githubusercontent.com/6329259/131214405-b9c1d17d-df75-45aa-91d4-a821f544a1a0.png)

**Comment by erictraut** [comment]
First, we should figure out why pylance is consuming so much memory. The only reason pylance would theoretically consume more memory is because of its symbol indexing used to improve auto completions and performance of "find all references". Pyright doesn't include this indexing capability. If you find that it's consuming too much memory or CPU in pylance, you can disable it in your settings.

```
"python.analysis.indexing": false
```

If that doesn't address the memory issue you're seeing with pylance, please report a bug in the [pylance-release issue tracker](https://github.com/microsoft/pylance-release/issues) so we can investigate further.

If you really want to use the pyright extension rather than pylance, you can set the "pyright.disableLanguageServices" setting to true. This will disable pyright's language server features including hover and auto completion.
